_toc.yml

```diff
@@ -341,7 +340,6 @@ entries:
             - file: docs/products/kafka/kafka-connect/howto/debezium-source-connector-sql-server
             - file: docs/products/kafka/kafka-connect/howto/debezium-source-connector-mongodb
             - file: docs/products/kafka/kafka-connect/howto/cassandra-streamreactor-source
-            - file: docs/products/kafka/kafka-connect/howto/mqtt-source-connector
```

Comment:
Why is this entry removed from `_toc`?

Reply:
I do not remember that I removed this entry. Only thing I did for this doc is adding a new entry. Let me add back.